src/components/useList/components/ListContainerView/ListContainerView.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import type {QAProps} from 'src/components/types';
+
+import {Flex} from '../../../layout';
+import {block} from '../../../utils/cn';
+
+import './ListContainerView.scss';
+
+const b = block('list-container-view');
+
+export interface ListContainerViewProps extends QAProps, React.HTMLAttributes<'div'> {
```

Comment:
Let's apply rest props via separate prop, `extraProps`. The component should alse allow to override `div` element